repo/init.go

```diff
@@ -168,6 +168,45 @@ func addConfigExtensions(repoRoot string, testnet bool) error {
 			LowFeeDefault:    20,
 			TrustedPeer:      "",
 		}
```

Comment:
I assume the `WalletConfig` above here will be deprecated? What's your plan with this?

Reply:
Yeah when the actual multiwallet is merged in that one can be removed.